infra/project-config/aws_services.tf

```diff
@@ -19,6 +19,7 @@ locals {
     "kms",
     "lambda",
     "logs",
+    "mobiletargeting",
```

Comment:
mobiletargeting == Pinpoint

Reply:
thanks, can we add it as an inline comment